src/host/libpixyusb_swig/image_player.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/python
```

Comment:
This script is just for demonstration purposes now. I have created a ticket to enhance this tool for viewing images off of the sd card via USB:  https://matternet.atlassian.net/browse/AVX-213